src/Console/Command/AddPrefixCommand.php

```diff
@@ -37,19 +38,23 @@
     const OUTPUT_DIR_OPT = 'output-dir';
     /** @internal */
     const FORCE_OPT = 'force';
+    /** @internal */
+    const REPLACE_STRINGS_OPT = 'replace-strings';
 
     private $fileSystem;
     private $handle;
+    private $stringReplacer;
 
     /**
      * @inheritdoc
      */
-    public function __construct(Filesystem $fileSystem, HandleAddPrefix $handle)
+    public function __construct(Filesystem $fileSystem, HandleAddPrefix $handle, StringReplacer $stringReplacer = null)
```

Comment:
`public function setReplaceMap(array $replaceMap)` => instead of using a mutator that way, I would just pass it to the handler. Not sure if we should encapsulate that into a Value Object to make it clearer, but IMO that would be cleaner and less confusing to pass it as a parameter